cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller.go

```diff
@@ -33,16 +37,18 @@ const (
 	nodeProviderIDIndex = "clusterapi-nodeProviderIDIndex"
 )
 
-// machineController watches for Nodes, Machines and MachineSets as
-// they are added, updated and deleted on the cluster. Additionally,
-// it adds indices to the node informers to satisfy lookup by
-// node.Spec.ProviderID.
+// machineController watches for Nodes, Machines, MachineSets and
+// MachineDeployments as they are added, updated and deleted on the
+// cluster. Additionally, it adds indices to the node informers to
+// satisfy lookup by node.Spec.ProviderID.
 type machineController struct {
-	clusterInformerFactory clusterinformers.SharedInformerFactory
-	kubeInformerFactory    kubeinformers.SharedInformerFactory
-	machineInformer        clusterv1alpha1.MachineInformer
-	machineSetInformer     clusterv1alpha1.MachineSetInformer
-	nodeInformer           cache.SharedIndexInformer
+	clusterClientset          clusterclient.Interface
```

Comment:
nit: I think clusterAPIInformerFactory clusterAPIClientSet would be more accurate

Reply:
> nit: I think `**clusterAPI**InformerFactory` and `**clusterAPI**ClientSet` would be more accurate

yep, will do. and check other cases/usages too.

Reply:
Done in https://github.com/openshift/kubernetes-autoscaler/pull/26/commits/7c704930c17e4ccfa91f250b45c32e47de78abe1